resources/puppetlabs/lein-ezbake/template/pe/ext/redhat/init.suse.erb

```diff
@@ -84,16 +75,13 @@ start() {
     # startproc will change users, so make sure that user has permission
     # to access the present working directory.
     cd "${INSTALL_DIR}"
-    startproc -u "${USER}" -l "${LOGFILE}" -p "${PIDFILE}" "/opt/puppetlabs/server/apps/${realname}/bin/${realname} start >> /var/log/puppetlabs/${realname}/${realname}-daemon.log 2>&1"
+    startproc -u "${USER}" -l "${LOGFILE}" -p "${PIDFILE}" -w -- "${INSTALL_DIR}/bin/${realname}" start >/dev/null 2>&1
```

Comment:
Cool, that's what I'd like to do as well.  Curious if @haus or @shrug in particular has a strong opinion on this point.